index.js

```diff
@@ -1,4 +1,4 @@
-/**
+	/**
```

Comment:
What is that? Fuck what for am I paying to you???

Reply:
This is unprofessional. I will report you to Vladimir.